site/src/pages/WorkspacePage/WorkspaceReadyPage.tsx

```diff
@@ -100,13 +100,11 @@ export const WorkspaceReadyPage = ({
       ["canceling", "deleting", "pending", "starting", "stopping"].includes(
```

Comment:
Random error I don’t understand. It happens when you start the stopped workspace.

<img width="1823" alt="Screenshot 2023-07-17 at 11 08 00" src="https://github.com/coder/coder/assets/14044910/46eca2a9-6d1c-4005-8864-76ea4e038072">

Reply:
Not able to reproduce... are you able to reproduce this error constantly?

Reply:
I can reproduce it every time.

Steps:
1. Create a new workspace with ephemeral parameter.
2. Stop the workspace.
3. "Start with options" the workspace.
4. Stop it again.
5. "Start" using the standard method (no options).

Reply:
I found it! 🙏